src/components/site/BackToTopFAB.tsx

```diff
@@ -0,0 +1,43 @@
+import { SVGProps, useEffect, useState } from "react"
+
+import { FABBase } from "../ui/FAB"
+
+function BxBxsArrowToTop(props: SVGProps<SVGSVGElement>) {
+  return (
+    <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}>
+      <path d="M6 4h12v2H6zm5 10v6h2v-6h5l-6-6l-6 6z" fill="currentColor" />
+    </svg>
+  )
+}
+
+const isShouldShow = () =>
+  document.documentElement.scrollTop > document.documentElement.clientHeight
+
+export const BackToTopFAB: React.FC<{}> = () => {
+  const [shouldShow, setShouldShow] = useState(isShouldShow())
+  useEffect(() => {
+    const handler = () => {
+      setShouldShow(isShouldShow())
```

Comment:
Hmm, maybe we don't need to listen to the scroll changes immediately. So, we can add a throttle wrapper to this function to reduce the state changes.